lib/extensions.js

```diff
@@ -28,10 +30,26 @@ function getRuntimeInfo() {
  */
 
 function getBinaryIdentifiableName() {
+  var v8SemVersion = process.versions.v8;
```

Comment:
You're correct it should be `(0,3)`

> I thought it would be good to save a join when v8 version already have three portions.

Since this is essentially a one off operation I don't see any benefit in saving a `join` when it comes at the cost of added verbosity.

It took me a couple reads of this this code to understand it and I already knew what it was trying to do.